modules/oadp-release-notes-1-1-4.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-4_{context}"]
+= OADP 1.1.4 release notes
+
+The OADP 1.1.4 release notes lists any new features, resolve issues and bugs, and known issues.
+
+[id="new-features_{context}"]
+== New features
+
+This version of OADP is a service release. No new features are added to this version.
+
+[id="resolved-issues_{context}"]
+== Resolved issues
+
+The issues and bugs that have been resolved in this version are listed in link:https://issues.redhat.com/browse/OADP-1398?jql=project%20%3D%20OADP%20AND%20status%20in%20(Closed%2C%20Verified%2C%20%22Release%20Pending%22)%20AND%20priority%20in%20(Blocker%2C%20Critical%2C%20Major)%20AND%20fixVersion%20in%20(%22OADP%201.1.4%22%2C%20%22OADP%201.1.4%22)%20AND%20assignee%20not%20in%20(rhn-support-hvider%2C%20rhn-support-anarnold%2C%20rhn-support-cwisemon%2C%20richard.hoch%2C%20rhn-support-sbeskin)%20ORDER%20BY%20description%20ASC[this page].
+
+[id="known-issues_{context}"]
+== Known issues
+
+The issues and bugs that are known in this version are listed in link:https://issues.redhat.com/browse/OADP-1619?jql=project%20%3D%20OADP%20AND%20status%20not%20in%20(Closed%2C%20Verified%2C%20%22Release%20Pending%22)%20AND%20priority%20in%20(Blocker%2C%20Critical%2C%20Major)%20AND%20fixVersion%20in%20(%22OADP%201.1.4%22%2C%20%22OADP%201.1.4%22)%20AND%20assignee%20not%20in%20(rhn-support-hvider%2C%20rhn-support-anarnold%2C%20rhn-support-cwisemon%2C%20richard.hoch%2C%20rhn-support-sbeskin)%20ORDER%20BY%20description%20ASC[this page].
```

Comment:
why would we include the release task here? @rayfordj wdyt..  is that needed?

Reply:
@weshayutin @rayfordj

Apologies, I will come back and do a re-write later tonight. There are also build errors which we need to fix.

Reply:
Thanks! :sunglasses: